coredb-operator/src/controller.rs

```diff
@@ -45,6 +48,8 @@ pub static COREDB_FINALIZER: &str = "coredbs.coredb.io";
 pub struct CoreDBSpec {
     #[serde(default = "defaults::default_replicas")]
     pub replicas: i32,
+    #[serde(default = "defaults::default_postgres_exporter_enabled")]
+    pub postgresExporterEnabled: bool,
```

Comment:
Would we ever want to configure the `postgresExporter`? If yes, we could do something like:
```
spec:
  postgresExporter:
    enabled: true
    config: value
```